tests/Fixtures/inc/Engine/CriticalPath/ProcessorService/processGenerate.php

```diff
@@ -36,11 +36,12 @@
 				'cpcss_exists_after' => false,
 				'request_timeout'    => true,
 				'mobile'             => false,
+				'type'               => 'post'
 			],
 			'expected' => [
 				'success' => false,
 				'code'    => 'cpcss_generation_timeout',
-				'message' => 'Critical CSS for http://example.org/?p=21 timeout. Please retry a little later.',
+				'message' => 'Critical CSS for post timeout. Please retry a little later.',
 				'data'    => [
 					'status' => 400,
 				],
```

Comment:
Let's add other types to the fixture too to ensure it behaves as expected.

Reply:
Done, now I added for homepage and category pages.
it needed some adjustments in tests and fixtures